data/mods/Magiclysm/ui/sidebar.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "id": "mana_num",
+    "type": "widget",
+    "label": "Mana",
+    "style": "number",
+    "var": "mana",
+    "colors": [ "c_red", "c_light_red", "c_yellow", "c_light_blue" ]
```

Comment:
The original mana widget was red (below 30%), yellow (below 50%) and light blue. There's room to be more imaginative with the colors here.

Reply:
I thought of something with more emphasis on blue, but the darkest blue is very hard to read on black. Something like `c_light_cyan`, `c_cyan`, `c_light_blue` could work but mana has always been `c_light_blue` when full and I don't want to mess with that.

So perhaps different colors are a subject for a future PR.